test/rules/APIGW.test.ts

```diff
@@ -183,42 +188,29 @@ describe('Amazon API Gateway', () => {
   describe('APIGWAuthorization: APIs implement authorization', () => {
     const ruleId = 'APIGWAuthorization';
     test('Noncompliance 1', () => {
-      new RestApi(stack, 'RestApi').root.addMethod('ANY');
+      new RestApi(stack, 'rRestApi').root.addMethod('ANY');
```

Comment:
please take out the `r` this is something we're trying to remove in the code base